### PR TITLE
Camera switch

### PR DESCRIPTION
**Pull request by Mixss**

### CODE REVIEW

File: src/mainboard/clevo/tgl-u/ramstage.c [hunk 1]
```diff
@@ -30,6 +30,10 @@ struct smfi_cmd_set_fan_curve {
 	struct fan_curve curve;
 } __packed;
 
+struct smfi_cmd_camera_enablement_set {
+    bool enable;
+} __packed;
```
Comment (mkopec) [inline src/mainboard/clevo/tgl-u/ramstage.c:35]: Does this need to be `__packed`?

Comment (miczyg1) [reply src/mainboard/clevo/tgl-u/ramstage.c:35]: Single element does not even need a structure...

File: src/mainboard/clevo/tgl-u/ramstage.c [hunk 2]
```diff
@@ -107,11 +116,46 @@ static void set_fan_curve(void)
...
+    if (ret != CB_SUCCESS) {
+		printk(BIOS_DEBUG, "camera: failed to read camera enablement status from EFI vars, using board default\n");
+		goto efi_err;
+    }
+
+efi_err:
+#endif
+
+    cmd.enable = enabled;
+    system76_ec_smfi_cmd(CMD_CAMERA_ENABLEMENT_SET, sizeof(cmd) / sizeof(uint8_t), (uint8_t *)&cmd);
```
Comment (miczyg1) [inline src/mainboard/clevo/tgl-u/ramstage.c:148]:
```suggestion
    system76_ec_smfi_cmd(CMD_CAMERA_ENABLEMENT_SET, sizeof(enabled), (uint8_t *)&enabled);
```

This should be enough

Comment (miczyg1) [reply src/mainboard/clevo/tgl-u/ramstage.c:148]: Then we can also get rid off `struct smfi_cmd_camera_enablement_set`

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Comment by miczyg1** [comment]
Please @Mixss use correct indentation...

**Pull request closed without merging**